packages/core/theme.ts

```diff
@@ -70,9 +70,15 @@ export const theme = {
       cyberpunk: '#a51aff',
       hyperlink: '#0000ea',
     },
+    height: {
```

Comment:
Should we be extending [`spacing`](https://tailwindcss.com/docs/customizing-spacing#extending-the-default-spacing-scale) rather than height and width directly?

Reply:
Ok yes thank you, I though the extend was “sizing” for some reason and couldn’t find it. Thought it was weird I had to declare both.